### ISSUE TITLE
Example.py results in "No Rows to Show"

### ISSUE THREAD

**Issue by kavinstewart** [ask-fini/paramount#6]
[paramount_data.csv](https://github.com/ask-fini/paramount/files/15046344/paramount_data.csv)
![image](https://github.com/ask-fini/paramount/assets/1700548/f09c567a-935d-4446-aa06-7090450eb523)

I'm using the default paramount.toml generated by running example.py:
```
[record]
enabled = true
fire_and_forget = false
replay_enabled = true
function_url = "http://localhost:9000"

[db]
type = "csv"

[api]
endpoint = "http://localhost"
port = 9001
split_by_id = false
identifier_colname = "paramount__recording_id"

[ui]
meta_cols = [ "recorded_at",]
input_cols = []
output_cols = []

[db.postgres]
connection_string = ""
```

**Comment by mrcn04** [comment]
Fixed with #7.

**Comment by kavinstewart** [comment]
I cloned the repo and ran the server using the script below (copied from the paramount bin)

```
import re
import sys
from paramount.server.cli import main
if __name__ == '__main__':
    sys.argv[0] = re.sub(r'(-script\.pyw|\.exe)?$', '', sys.argv[0])
    sys.exit(main())
```
    
Got the same problem though (No Rows to Show)

**Comment by hakimkhalafi** [comment]
Hi @kavinstewart - this has been fixed now with 0.4.0. 

For next steps, remember to set the **chat_list**, **chat_list_role_param** and **chat_list_content_param** settings in paramount.toml to see actual chat, as suggested by the README.md

Let us know if you run in to any other issues, and feel free to ping me for further help in terms of setting this up for your organization.
